data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -701,49 +714,100 @@
+                    [/message]
+                    [message]
+                        speaker=Rugnur
+                        message= _ "Wait! If Krawg can escape, shouldn’t we give him the Sceptre?"
+                    [/message]
+                    [message]
+                        speaker=Thursagan
+                        message= _ "No, too risky, even if Krawg drops it in King Haldric’s hand. You’ve seen how the elves have behaved, you’ve seen how your lord behaved. The corrupting emissions from the ruby are nothing compared to the greed already out there, and my crafted gold cage does nothing to contain that."
+                    [/message]
+                    [message]
+                        speaker=Thursagan
+                        message= _ "Durstorn and Glildur have both convinced me that the time is not right for the Sceptre of Fire, best that it rests with us for a while. Now, Krawg!"
+                    [/message]
+                    [message]
+                        speaker=Krawg
+                        message= _ "Yah! Krah!"
+                    [/message]
+                [/then]
+                [else]
+                    [message]
+                        speaker=$unit.id
+                        message= _ "There’s a tablet with a rune here!"
+                    [/message]
+                    [message]
+                        speaker=Thursagan
+                        message= _ "Yes, that must be the last rune sealing the volcano. Smash it, like the others!"
+                    [/message]
+                    [message]
+                        speaker=$unit.id
+                        message= _ "The resulting eruption will lead to all of your deaths!"
+                    [/message]
+                    [message]
+                        speaker=Thursagan
+                        message= _ "Just do it! You riders have a chance to escape, our enemies cannot. We knew that death was likely when we fled here, just go!"
+                    [/message]
+                    [message]
+                        speaker=Rugnur
+                        message= _ "Wait! If the riders can escape, shouldn’t we give them the Sceptre?"
+                    [/message]
+                    [message]
+                        speaker=Thursagan
+                        message= _ "No, too risky, even if they drop it in King Haldric’s hand. You’ve seen how the elves have behaved, you’ve seen how your lord behaved. The corrupting emissions from the ruby are nothing compared to the greed already out there, and my crafted gold cage does nothing to contain that."
+                    [/message]
+                    [message]
+                        speaker=Thursagan
+                        message= _ "Durstorn and Glildur have both convinced me that the time is not right for the Sceptre of Fire, best that it rests with us for a while. Now, smash it!"
```

Comment:
Late comment, because I only realised when reviewing the .po file fuzzies:
```
# po: The "smash it" refers to a stone tablet, not the Sceptre.
```

Reply:
Thanks for handling this in #6633.